docs/pages/sass/+Page.mdx

```diff
@@ -1,6 +1,6 @@
 import { Example, CommunityNote } from '../../components'
 
-<CommunityNote />
+<CommunityNote url="https://sass-lang.com" />
 
 For using [Sass](https://sass-lang.com)/[Less](https://lesscss.org)/[Stylus](https://stylus-lang.com), all you have to do is to install the CSS pre-processor you want to use: Vite will then automatically integrate it. See [Vite Docs - CSS pre-processors](https://vitejs.dev/guide/features.html#css-pre-processors).
```

Comment:
Duplicated link. Intervention++. (It's much slower for me to review when I have to review details, that's why I'd be great if we minimize these kind of interventions.)

Reply:
Ah, I see why you kept it. Nevermind.